tooling-e2e-tests/scheduler_scenario.sh

```
@@ -0,0 +1,114 @@
+scenario() {
+  configure_cli ${1} ${2}
+  log_progress "✅ CLI configured"
+
+  relay_after_seconds=5
+  withdrawal_amount=$(mtzero 50)
+
+  #####################################################################################
+  ################################# Native withdrawal #################################
+  #####################################################################################
+  relayer_fee_quote=$(quote_relayer_fee)
+  max_relayer_fee=$(($relayer_fee_quote / 10 + $relayer_fee_quote)) # 110% of the quoted fee
```

Shellcheck (0.11.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)